File: test/fixtures/kotlin/get_user_agent_alias.kt
import java.io.IOException
import okhttp3.OkHttpClient
import okhttp3.Request

val client = OkHttpClient()

val request = Request.Builder()
  .url("http://localhost:28139/vc/moviesmagic?p=5&pub=testmovie&tkn=817263812")
  .header("x-msisdn", "XXXXXXXXXXXXX")
  .header("user-agent", "Mozilla Android6.1")
  .build()

client.newCall(request).execute().use { response ->
  if (!response.isSuccessful) throw IOException("Unexpected code $response")
  response.body!!.string()
}

File: test/fixtures/r-httr2/get_user_agent_alias.r
library(httr2)

request("http://localhost:28139/vc/moviesmagic") |>
  req_url_query(
    p = "5",
    pub = "testmovie",
    tkn = "817263812"
  ) |>
  req_headers(
    `x-msisdn` = "XXXXXXXXXXXXX",
    `user-agent` = "Mozilla Android6.1"
  ) |>
  req_perform()
